### ISSUE TITLE
Where does virtual filesystem appear?  Does it block access to the real filesystem?

### ISSUE THREAD

**Issue by cspotcode** [nexe/nexe#736]
I couldn't find this in the docs.  When I bundle a `node_modules` directory into a `nexe` binary, and I need to `fs.readFile()` a bundled file, what is the absolute path to that file?

**Comment by calebboyd** [comment]
The file system is captured (includes everything that is statically analyzable and included in --resources) relative to the `cwd` at compile time, and then rendered relative to the executable location.

It does block access to the real file system at this time. This is not ideal.

**Comment by cspotcode** [comment]
Dang, this is an immediately blocker.

Is it feasible for `nexe` to adopt `pkg`'s approach, where the virtual filesystem appears at `/snapshot`.  This blocks access to a `/snapshot` directory that might exist on the host machine, but that seems far better than blocking access to the `node_modules` directory within cwd.

For my use-case, I write reusable CLI tools.  I bundle `node_modules` into the `nexe` binary.  Supposing I write a testing tool that is bundled by `nexe`, and my tool needs to access a shared config that lives in the consumer's `node_modules` directory.  My tool now needs to access the contents of `node_modules`.

Supposing my tool is configurable with a `my-tool.config.js` file, similar to eslint, prettier, tsconfig, etc.

```
module.exports = {
    extends: "shared-config-published-on-npm",
    flag: false
}
```

My tool will read `./my-tool.config.js`, then try to read `./node_modules/shared-config-published-on-npm/index.js`.  I assume this will fail in a confusing way?

**Comment by calebboyd** [comment]
Makes sense. I don't have an immediate solution (or bandwidth to work on one right now) but if there is "the one thing to change" it is this.

**Comment by cspotcode** [comment]
Looks like I can change how `baseDir` is computed; hard-code it to `/snapshot` or `C:\snapshot`
https://github.com/nexe/nexe/blob/master/src/fs/patch.ts#L38

**Comment by calebboyd** [comment]
Hmm If that works let me know
